frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -913,41 +942,232 @@ function ReaderFooter:addToMainMenu(menu_items)
+                    {
+                        text = _("Set size"),
+                        separator = true,
+                        callback = function()
+                            local value, value_min, value_max
+                            if self.settings.progress_style_thin then
+                                value = self.settings.progress_style_thin_height or 3
+                                value_min = 1
+                                value_max = 4
+                            else
+                                value = self.settings.progress_style_thick_height or 7
+                                value_min = 5
+                                value_max = 14
+                            end
+                            local SpinWidget = require("ui/widget/spinwidget")
+                            local items = SpinWidget:new{
+                                width = Screen:getWidth() * 0.6,
+                                value = value,
+                                value_min = value_min,
+                                value_step = 1,
+                                value_hold_step = 2,
+                                value_max = value_max,
+                                title_text =  _("Progress bar size"),
+                                callback = function(spin)
+                                    if self.settings.progress_style_thin then
+                                        self.settings.progress_style_thin_height = spin.value
+                                    else
+                                        self.settings.progress_style_thick_height = spin.value
+                                    end
+                                    self:refreshFooter()
+                                    self.ui:handleEvent(Event:new("SetPageBottomMargin", self.view.document.configurable.b_page_margin))
+                                    UIManager:setDirty(nil, "ui")
+                                end
+                            }
+                            UIManager:show(items)
+                        end
+                    },
+                    {
+                        text = _("Show chapter markers"),
+                        checked_func = function()
+                            return self.settings.toc_markers
+                        end,
+                        enabled_func = function()
+                            return not self.settings.progress_style_thin
+                        end,
+                        callback = function()
+                            self.settings.toc_markers = not self.settings.toc_markers
+                            self:setTocMarkers()
+                            self:updateFooter()
+                            UIManager:setDirty(nil, "ui")
+                        end
+                    },
+                },
+            },
+            {
+                text_func = function()
+                    local pos = _("balanced (10)")
```

Comment:
I would *tend* to prefer "centered" over "balanced", but let's wait for @Frenzie for a comment on that ;).

Reply:
What exactly does it mean?

Reply:
With a fixed L+R margin.

Reply:
Centered sounds like the right term for that, yes.

Reply:
But "like book margins" is centered too.
This "Balanced" item means centered, but with fixed margins, with some default fixed margins values that do please most people (so, statistically balanced :)

Reply:
Static margins?